next/components/forms/ProgressBar.tsx

```diff
@@ -0,0 +1,35 @@
+import cx from 'classnames'
+import {useProgressBar} from 'react-aria'
+
+type ProgressBarBase = {
+  type?: 'success' | 'default'
+  value: number
+  minValue?: number
+  maxValue?: number
+  className?: string
+}
+
+const ProgressBar = ({ type = 'default', value = 0, minValue = 0, maxValue = 100, className }:ProgressBarBase) => {
+  const {
+    progressBarProps
+  } = useProgressBar({value,minValue,maxValue});
+
+  const percentage = (value - minValue) / (maxValue - minValue);
+  const barWidth = `${Math.round(percentage * 100)}%`;
+
+  const progressBarStyleContainer = cx('flex flex-row items-center p-0 gap-4 w-full max-w-[500px] h-6', className, {
```

Comment:
I don't think max-w should be hardcoded inside the component - we want the 'user' of the component to determine the width of this, i.e. by controlling the enclosing component

That is, I think you can just remove the max-widths from this and it should be reasonable ? though you may need to do more with the inner div below to make it behave as expected